CHANGELOG.rst

```diff
@@ -2,6 +2,199 @@
 Release Notes
 =============
 
+.. _Release Notes_6.5.0:
+
+6.5.0
+=====
+
+.. _Release Notes_6.5.0_Prelude:
+
+Prelude
+-------
+
+Release on: Unreleased 
+
+- Please refer to the `6.5.0 tag on integrations-core <https://github.com/DataDog/integrations-core/releases/tag/6.5.0>`_ for the list of changes on the Core Checks.
+
+- Please refer to the `6.5.0 tag on trace-agent <https://github.com/DataDog/datadog-trace-agent/releases/tag/6.5.0>`_ for the list of changes on the Trace Agent.
+
+- Please refer to the `6.5.0 tag on process-agent <https://github.com/DataDog/datadog-process-agent/releases/tag/6.5.0>`_ for the list of changes on the Process Agent.
+
+
+.. _Release Notes_6.5.0_New Features:
+
+New Features
+------------
+
+- Autodiscovery: the ``docker`` and ``kubelet`` listeners will retry on error,
+  to support starting the agent before your container runtime (host install)
+
+- Bump the default number of check runners to 4. This has some
+  concurrency implications as we will now run multiple checks in
+  parallel.
+
+- Kubernetes: to avoid hostname collisions between clusters, a new ``cluster_name`` option is available. It will be added as a suffix to the host alias detected from the kubelet in order to make these aliases unique across different clusters.
+
+- Docker image: handle docker/kubernetes secret files with a helper script
+
+- The Node Agent can rely on the Datadog Cluster Agent to collect Node Labels.
+
+- Improved ECS fargate tagging:
+  - Honor the ``docker_labels_as_tags`` option to extract custom tags
+  - Make the ``cluster_name`` tag shorter
+  - Add the ``short_image`` and ``container_id`` tags
+  - Remove some noisy tags
+  - Fix a lifecycle issue that caused missing tags
```

Comment:
format of this list is incorrect in `rst`, should be fixed

Reply:
Nested list with `*` - tested here: http://www.siafoo.net/reST.xml